model/src/test/java/org/projectnessie/model/TestModelObjectsSerialization.java

```diff
@@ -47,7 +49,15 @@ public class TestModelObjectsSerialization {
   @MethodSource("goodCases")
   void testGoodSerDeCases(Case goodCase) throws IOException {
     String json = MAPPER.writeValueAsString(goodCase.obj);
-    Assertions.assertThat(json).isEqualTo(goodCase.deserializedJson);
+    JsonNode j;
```

Comment:
String comparison is flaky/error-prone, because the actual order of elements might change.
Parsing the strings as `JsonNode`s is safe though.